app/models/comment.rb

```diff
@@ -2,9 +2,21 @@ class Comment < ApplicationRecord
   belongs_to :post
   validates_presence_of :body
 
+  def self.best(comments)
```

Comment:
What do you think about using the rank_order method here to eliminate duplication?